charactersheet/charactersheet/templates/spells.tmpl.html

```diff
@@ -52,8 +52,15 @@
           <!-- /ko -->
           <td data-bind="text: spellName, click: $parent.editSpell"
                href="#"></td>
+
+          <!-- ko if: spellLevel() != 0 -->
```

Comment:
@tanyxp sounds good, I'm going to make a ticket to add this blurb to the 'View Model Programming Guide'